source/js/components/people/people.scss

```diff
@@ -9,14 +9,12 @@
   }
 
   .person-card-featured {
-    .more-details {
-      background-color: $blue;
-      color: white;
+    
 
       &:hover {
```

Comment:
this has now changed from being `.person-cared-featured .more-details:hover` to `.person-cared-featured:hover`. If nothing uses `.more-details` then its associated pseudo rules should not be kept.

Reply:
My bad,
working on it. 👍